unity-client/Assets/Scripts/MainScripts/DCL/Components/Avatar/AvatarModifiers/AvatarModifierArea.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
+using DCL;
+using DCL.Components;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class AvatarModifierArea : BaseComponent
+{
+    [Serializable]
+    public class Model
+    {
+        // TODO: Change to TriggerArea and handle deserialization with subclasses
+        public BoxTriggerArea area;
```

Comment:
JSON deserialization doesn't detect subclasses correctly. So I set the only subclass we have right now. If/when we decide to support other areas, we will need to change this